pandas/core/frame.py

```diff
@@ -4525,7 +4525,7 @@ def set_index(
         append : bool, default False
             Whether to append columns to existing index.
         inplace : bool, default False
-            Modify the DataFrame in place (do not create a new object).
+            If set to True, modifies the DataFrame in place (do not create a new object).
```

Comment:
@PurushothamanSrikanth this line needs to be wrapped to make the linter pass (or remove "set to" might do it)